src/screens/SendBitcoin/SendBitcoinConfirm.js

```diff
@@ -99,22 +118,28 @@ class SendBitcoinConfirm extends React.Component<Props, State> {
         >
           <LabeledRow>
             <Label>Amount</Label>
-            <Value>{amount} BTC</Value>
+            <Value>{amount} BTC ({formattedAmountInFiat})</Value>
           </LabeledRow>
           <LabeledRow>
             <Label>Recipient Address</Label>
             <Value>{to}</Value>
           </LabeledRow>
+          <LabeledRow>
+            <Label>Total fee</Label>
+            <Value>{feeInBtc} BTC ({formattedFeeInFiat})</Value>
```

Comment:
<Value>{formattedFeeInBtc} BTC ({formattedFeeInFiat})</Value> could be added for better view